source/DSCResources/DSC_CMEmailNotificationComponent/DSC_CMEmailNotificationComponent.psm1

```diff
@@ -0,0 +1,434 @@
+$script:dscResourceCommonPath = Join-Path -Path $PSScriptRoot -ChildPath '..\..\Modules\DscResource.Common'
+$script:configMgrResourcehelper = Join-Path -Path $PSScriptRoot -ChildPath '..\..\Modules\ConfigMgrCBDsc.ResourceHelper'
+
+Import-Module -Name $script:dscResourceCommonPath
+Import-Module -Name $script:configMgrResourcehelper
+
+$script:localizedData = Get-LocalizedData -DefaultUICulture 'en-US'
+
+<#
+    .SYNOPSIS
+        This will return a hashtable of results.
+
+    .PARAMETER SiteCode
+        Specifies the SiteCode for the Configuration Manager site.
+
+    .PARAMETER Enabled
+        Specifies if email notifications are enable or disable.
```

Comment:
Let's say "enabled or disabled" here, line 66, 266, in the readme, and in the schema.